libraries/standard/http/cbmc/proofs/HTTPClient_Send/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=HTTPClient_Send_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY += __CPROVER_file_local_http_client_c_findHeaderFieldParserCallback
```

Comment:
Did we decide to do this functions separately?

Reply:
I think the point is that these functions are removed because they are not actually reachable, and removing them makes the coverage count more accurate.

Reply:
I will put in the backlog that we need to prove these functions separately.